dev/WindowsAppSDK_Insights/WindowsAppSDKInsightsLib/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.210803.1" targetFramework="native" />
-</packages>
```

Comment:
do we still need nuget.config, packages.config?

Reply:
This file is deleted. We don't need this packages.config and nuget.config. The one in the root folder is still needed.